app/src/routes/login/components/sso-links.vue

```diff
@@ -21,25 +30,28 @@ export default defineComponent({
 	setup() {
 		const { t } = useI18n();
 
-		const providers = ref([]);
+		const authProviders = ref([]);
+		const oauthProviders = ref([]);
 		const loading = ref(false);
 
 		onMounted(() => fetchProviders());
 
-		return { t, providers };
+		return { t, authProviders, oauthProviders };
 
 		async function fetchProviders() {
 			loading.value = true;
 
 			try {
-				const response = await api.get('/auth/oauth/');
+				const [authResponse, oauthResponse] = await Promise.all([api.get('/auth/'), api.get('/auth/oauth/')]);
+
+				authProviders.value = authResponse.data.data?.map((providerName: string) => ({
+					name: providerName,
```

Comment:
It would be nice if this (and oauth) had underscores parsed out and title formatting.